### PR TITLE
clarified note about what an activity is

### LINKED ISSUE #2351: Note about what an activity is is likely to be wrong
Hello,

I think the statement in the note about what an activity is is wrong.

`Activity is counted as requests sent to SharePoint, such as mouse clicks.  Moving the mouse and scrolling are not counted as activity.`

There are no requests sent to the SharePoint by just clicking; a mouse click has to happen on a button or link to send a request as mentioned in the note, I think. That would make more sense because moving the mouse and scrolling isn't counted too. So clicking on an active element to trigger a link open or an action counts as activity.

Most likely the person who wrote this was right about how it works. Just the wording is a bit unclear.

Thank you for checking this.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cfaf9fb8-afec-06b3-ed85-244a7af5b6dd
* Version Independent ID: 016c6f7d-8fd1-ba7f-ab54-a0ef1d34d026
* Content: [Sign out inactive users - SharePoint in Microsoft 365](https://docs.microsoft.com/en-us/sharepoint/sign-out-inactive-users)
* Content Source: [SharePoint/SharePointOnline/sign-out-inactive-users.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/sign-out-inactive-users.md)
* Service: **sharepoint-online**
* GitHub Login: @kaarins
* Microsoft Alias: **kaarins**

### PR DESCRIPTION
**Pull request by GiantCrocodile**
closes #2351

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/sign-out-inactive-users.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ When a user is inactive in SharePoint and OneDrive for a period of time you spec
 ![Inactive Microsoft 365 sign out warning message](media/80c0d10b-df78-4e3c-9df0-b94a923b3871.png)
   
 > [!NOTE]
-> Activity is counted as requests sent to SharePoint, such as mouse clicks.  Moving the mouse and scrolling are not counted as activity.
+> Activity is counted as requests sent to SharePoint, such as mouse clicks on an UI element like a button, list entry or a link.  Moving the mouse and scrolling are not counted as activity.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sign-out-inactive-users.md:43]:
```suggestion
> Activity is counted as requests sent to SharePoint, such as mouse clicks on a UI element like a button, list entry, or a link. Moving the mouse and scrolling are not counted as activity.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: SharePoint/SharePointOnline/sign-out-inactive-users.md (hunk 1 above)
Comment (GiantCrocodile) [reply SharePoint/SharePointOnline/sign-out-inactive-users.md:43]: Isn't 'an' correct in this case? Because of the U in UI.

Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/sign-out-inactive-users.md:43]: It depends on the sound of the first letter of the acronym when it is pronounced. If it is a vowel letter but it has a consonant sound, you should use a.

Comment (GiantCrocodile) [reply SharePoint/SharePointOnline/sign-out-inactive-users.md]: Thank you for the great explanation @JohanFreelancer9!

**Pull request merged**